### DISCUSSION TITLE
feature request: support di-birectional audio flow of Ring Intercom

### DISCUSSION THREAD

**Discussion by pilot1981** [koush/scrypted#740] [Ideas]
Hi,
I would ask if you can add support of bi-directional audio flow of Ring Intercom?

regards

**Comment by pilot1981** [comment]
any news?

**Comment by Vahn84** [comment]
+1

**Comment by bzumik1** [comment]
+1

**Comment by vifino** [comment]
I'd love to see this as well, given that this is basically the one distinguishing feature of the Ring Intercom compared to other products/projects for multi-tenant houses utilizing something like the Ritto Twinbus.
